e2e/terraform/packer/ubuntu-bionic-amd64.pkr.hcl

```diff
@@ -15,6 +15,7 @@ source "amazon-ebs" "latest_ubuntu_bionic" {
   instance_type        = "t2.medium"
   region               = "us-east-1"
   ssh_username         = "ubuntu"
+  ssh_interface        = "public_ip"
```

Comment:
This was required for me to successfully build e2e packer images locally, but I'm not sure what impact it will have to our automated builds.